third_party/bigframes_vendored/ibis/backends/sql/compilers/bigquery/__init__.py

```diff
@@ -0,0 +1,1106 @@
+# Contains code from https://github.com/ibis-project/ibis/blob/main/ibis/backends/sql/compilers/bigquery/__init__.py
+
+"""Module to convert from Ibis expression to SQL string."""
+
+from __future__ import annotations
+
+import decimal
+import math
+import re
+from typing import Any, TYPE_CHECKING
+
+from ibis import util
+from ibis.backends.sql.compilers.base import AggGen, NULL, SQLGlotCompiler, STAR
+from ibis.backends.sql.compilers.bigquery.udf.core import PythonToJavaScriptTranslator
+from ibis.backends.sql.datatypes import BigQueryType, BigQueryUDFType
+from ibis.backends.sql.rewrites import (
```

Comment:
Should we change this import to be our vendored copy? Or is that going to happen in a subsequent PR?

Reply:
Fixed in this PR. Thanks!